functions-and-operators/tidb-functions.md

```diff
@@ -257,3 +257,73 @@ Check Table Before Drop: false
 ### MySQL compatibility
 
 The `TIDB_VERSION` function is TiDB-specific and not compatible with MySQL. If MySQL compatibility is required, you can also use `VERSION` to get version information, but the result does not contain build details.
+
+## TIDB_DECODE_SQL_DIGESTS
+
+The `TIDB_DECODE_SQL_DIGESTS` function is used to query the normalized SQL statements (a form without format and parameter) corresponding to the set of SQL digests in the cluster. This function accepts 1 or 2 parameters:
+
+* `digests`: The string type, this parameter should conform to the format of a JSON string array, and each string in the array should be a SQL Digest.
```

Comment:
```suggestion
* `digests`: The string type. This parameter is in the format of a JSON string array, and each string in the array is a SQL digest.
```

Reply:
Looks kind of strange. the parameter `digest`'s type is string and `stmtTruncateLength`'s type is integer.

Reply:
This is what is Chinese version says: https://github.com/pingcap/docs-cn/pull/6824/files#diff-78df1cdc91d235cfa2c1cc93e95e4b307abb9ae13f9a668a053bda7560d9e551R268

Reply:
It means *this parameter is of xxx type*, rather than *this parameter is **the** xxx type*.